src/index.es6

```diff
@@ -0,0 +1,39 @@
+/*jshint esversion: 6 */
+/*
+ * This file is part of Cockpit.
```

Comment:
I think this is more of a sample project / starter kit. Should we get rid of this line altogether?